package.json

```diff
@@ -8,6 +8,7 @@
         "src",
         "!src/**/*.test.js"
     ],
+    "types": "dist/index.d.ts",
```

Comment:
I am not sure index.d.ts is what we want as main input. Actually, it didn't work for me when I tried with it. I had to use pencil.d.ts because that's where all the components are actually exported.

Reply:
I'll try again to confirm, though.

Reply:
The `index.js` is [exporting everything in `pencil.js`](https://github.com/pencil-js/pencil.js/blob/b402b4b452bcfeda236c29b892fdd2fe3686ce49/src/index.js#L29). I'm not experienced with TypeScript, so I'll let you confirm what works best.